app/Http/Controllers/ArticleController.php

```diff
@@ -88,14 +91,13 @@ public function edit(Article $article)
      */
     public function update(Request $request, Article $article)
     {
-        //フォームに入力された内容を変数に取得
+        $article = Article::find($request->id);
         $form = $request->all();
-
-        // フォームに入力された内容をデータベースへ登録
+        unset($form['_token']);
```

Comment:
```suggestion
        unset($form['_token']);
```

なんでtokenのunset必要なんやっけ?